addons/common/functions/fnc_registerItemReplacement.sqf

```diff
@@ -0,0 +1,115 @@
+#include "script_component.hpp"
+/*
+ * Author: BaerMitUmlaut
+ * Registers an event handler that replaces an item or item type with one or
+ * more other items.
+ *
+ * Arguments:
+ * 0: Item or item type ID to replace <STRING or NUMBER>
```

Comment:
what is "item type ID"?

Reply:
https://github.com/acemod/ACE3/blob/master/addons/main/script_macros.hpp#L63-L97

Reply:
Missed Line 47 I guess ^^